src/transformer/descriptor/descriptor.ts

```diff
@@ -46,6 +46,12 @@ import { GetShorthandPropertyAssignmentDescriptor } from './shorthandPropertyAss
 import { GetParameterDescriptor } from './parameter/parameter';
 import { GetVariableDeclarationDescriptor } from './variable/variable';
 
+const GetParenthesizedExpressionDescriptor: (
```

Comment:
is there a reason why this is not in its own file? I don't mind but I want to make sure it's not a mistake

Reply:
Just a mistake :) forgot to refactor!

Reply:
done